### PR TITLE
Character detail

### PR DESCRIPTION
**Pull request by fatmatanis**
[character detail page.webm](https://user-images.githubusercontent.com/97517723/188377821-fe529dbe-bb42-478a-bcde-bcbf400516f8.webm)

### CODE REVIEW

File: src/components/Navbar/styles.scss [hunk 1]
```diff
@@ -113,9 +113,9 @@
 
 @media (max-width: $breakpoint-md) {
     .search {
-        width: calc(100% - 32px);
+        width: calc(100% - 49px);
```
Comment (atvdi) [inline src/components/Navbar/styles.scss]: Why 49?

File: src/pages/CharacterDetail/index.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        { title: "Status", text: data.character.status },
+        { title: "Gender", text: data.character.gender },
+        { title: "Species", text: data.character.species },
+        { title: "Origin", text: data.character.origin.name },
+        {
+          title: "Type",
+          text: data.character.type === "" ? "Unknown" : data.character.type
+        },
+        { title: "Location", text: data.character.location.name }
+      ]);
```
Comment (atvdi) [inline src/pages/CharacterDetail/index.tsx:40]: Let's put this variable into a separate file (like `constants/index.ts`) and import it.

**Review by atvdi — COMMENTED** [review]
Nice work 👍 Added some comments

File: src/components/Navbar/styles.scss (hunk 1 above)
Comment (fatmatanis) [reply src/components/Navbar/styles.scss]: should have been 48, thanks :)

File: src/pages/CharacterDetail/index.tsx (hunk 2 above)
Comment (fatmatanis) [reply src/pages/CharacterDetail/index.tsx:40]: thanks, i will do this

**Review by atvdi — APPROVED** [review]

**Pull request merged**